extensions/devservices/h2/src/main/java/io/quarkus/devservices/h2/deployment/H2DevServicesProcessor.java

```diff
@@ -49,12 +53,12 @@ public RunningDevServicesDatasource startDatabase(Optional<String> username, Opt
                     LOG.info("Dev Services for H2 started.");
 
                     String connectionUrl = "jdbc:h2:tcp://localhost:" + tcpServer.getPort() + "/mem:"
-                            + datasourceName.orElse("default")
+                            + datasourceName.orElse(DEFAULT_DATABASE_NAME)
                             + ";DB_CLOSE_DELAY=-1" + additionalArgs.toString();
                     return new RunningDevServicesDatasource(null,
                             connectionUrl,
-                            "sa",
-                            "sa",
+                            DEFAULT_DATABASE_USERNAME,
+                            DEFAULT_DATABASE_PASSWORD,
```

Comment:
Just checking but we have the full permissions when doing that and not using `sa`? I thought it had a special meaning? Given I don't see us defining permissions for the `quarkus` user, I'm wondering how this works?

Reply:
> By default, if the database specified in the [embedded](https://h2database.com/html/features.html#database_url) URL does not yet exist, a new (empty) database is created automatically. The user that created the database automatically becomes the administrator of this database.

From H2 docu https://h2database.com/html/tutorial.html#creating_new_databases
`sa` is the abriviation for `System Administrator`, but seems to have no special permissions. It is just some default IMHO.